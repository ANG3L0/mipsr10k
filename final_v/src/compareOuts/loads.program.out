Chronologic VCS simulator copyright 1991-2011
Contains Synopsys proprietary information.
Compiler version E-2011.03_Full64; Runtime version E-2011.03_Full64;  Dec 10 00:51 2012
@@
@@
@@                     0  Asserting System reset......
@@
@@                     0 : System STILL at reset, can't show anything
@@
@@
@@                    10 : System STILL at reset, can't show anything
@@
@@
@@                    20 : System STILL at reset, can't show anything
@@
@@@ Unified Memory contents hex on left, decimal on right: 
@@@
@@@ mem[    0] = 203f100020bf0000 : 2323593525481897984
@@@ mem[    8] = a4610000b4410000 : 11844748497985273856
@@@ mem[   16] = 0000000000000555 : 1365
@@@
@@@
@@                   152 : System halted
@@
@@@ System halted on HALT instruction
@@@
@@
@@  14 cycles / 5 instrs = 2.800000 CPI
@@
@@  390.00 ns total time to execute
@@

$finish called from file "testbench/pipeline_test.v", line 817.
$finish at simulation time                  152
           V C S   S i m u l a t i o n   R e p o r t 
Time: 152
CPU Time:      0.290 seconds;       Data structure size:   0.3Mb
Mon Dec 10 00:51:21 2012
