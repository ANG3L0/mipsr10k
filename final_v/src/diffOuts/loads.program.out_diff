< @@@ mem[    0] = 205f0005203f0001 : 2332583154017042433
< @@@ mem[    8] = 209f0008207f0002 : 2350597565415620610
< @@@ mem[   16] = 4083052640830404 : 4648564902225708036
< @@@ mem[   24] = 40c4040640410522 : 4666859538756994338
< @@@ mem[   32] = 402604074c630404 : 4622386496935232516
< @@@ mem[   40] = f45ffffa4c640414 : 17609074518530458644
< @@@ mem[   48] = 0000055540410409 : 5863708361737
> @@@ mem[    0] = 203f100020bf0000 : 2323593525481897984
> @@@ mem[    8] = a4610000b4410000 : 11844748497985273856
> @@@ mem[   16] = 0000000000000555 : 1365
