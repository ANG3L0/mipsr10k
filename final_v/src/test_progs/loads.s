/*
	

*/
	data = 0x1000
	lda	$r5,0
	lda	$r1,data
	#loop:	mulq	$r5,0x0a,$r2
	stq	$r2,0($r1)
	ldq	$r3,0($r1)
	#stq     $r3,0x100($r1)
	#addq    $r1,0x8,$r1
	#addq	$r5,0x1,$r5
	#cmple   $r5,0xf,$r4
	#bne     $r4,loop
	call_pal        0x555

