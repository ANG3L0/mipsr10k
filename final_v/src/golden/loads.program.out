Chronologic VCS simulator copyright 1991-2012
Contains Synopsys proprietary information.
Compiler version G-2012.09_Full64; Runtime version G-2012.09_Full64;  Dec 10 00:19 2012
@@
@@
@@                     0  Asserting System reset......
@@
@@                     0 : System STILL at reset, can't show anything
@@
@@
@@                   100 : System STILL at reset, can't show anything
@@
@@
@@                   200 : System STILL at reset, can't show anything
@@
@@
@@                   300 : System STILL at reset, can't show anything
@@
@@                   360  Deasserting System reset......
@@
@@
@@@ Unified Memory contents hex on left, decimal on right: 
@@@
@@@ mem[    0] = 203f100020bf0000 : 2323593525481897984
@@@ mem[    8] = a4610000b4410000 : 11844748497985273856
@@@ mem[   16] = 0000000000000555 : 1365
@@@
@@@
@@                  4020 : System halted
@@
@@@ System halted on HALT instruction
@@@
@@
@@  37 cycles / 4 instrs = 9.250000 CPI
@@
@@  1080.00 ns total time to execute
@@

$finish called from file "testbench/testbench.v", line 290.
$finish at simulation time                 5020
           V C S   S i m u l a t i o n   R e p o r t 
Time: 502000 ps
CPU Time:      0.450 seconds;       Data structure size:   0.1Mb
Mon Dec 10 00:19:32 2012
